### PR TITLE
Add rules for commas and parentheses

### PR DESCRIPTION
**Pull request by SergioEstevao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,44 @@ guard condition else { return }
...
+```
+
+**Not Preferred:**
+```swift
+if (name == "Hello") {
+  print("World")
+}
+```
+
+In larger expressions, optional parentheses can sometimes make code read more clearly.
```
Comment (koke) [inline README.md]: Do you have any examples of this in the code? I can't think of a good exception and I'd rather remove this comment

Comment (diegoreymendez) [reply README.md]: I'm not sure I agree with the wording, but I agree with the idea we should aim for no optional parentheses.

This is an exaggeration, but it shows there's at least some truth in the statement above:

screen shot 2017-10-16 at 1 41 19 pm

File: README.md [hunk 2]
```diff
@@ -143,6 +143,44 @@ guard condition else { return }
 if condition { } else { }
 ```
 
+### Commas
+
+There should be no space before an comma and a single one after any comma.
```
Comment (diegoreymendez) [inline README.md]: Replace `an` with `a`.

File: README.md [hunk 3]
```diff
@@ -143,6 +143,44 @@ guard condition else { return }
...
+**Not Preferred:**
+```swift
+function print(name: String ,surname: String) {
+  ...
+}
+```
+
+## Parentheses
+
+Parentheses around conditionals, if,for,while,do statements, are not required and should be omitted.
```
Comment (diegoreymendez) [inline README.md]: For consistency with the rest of the documentation, add a space after the commas.

**Review by diegoreymendez — CHANGES_REQUESTED** [review]
Added a few inline comments.

I hate to be the one to point this out, but this PR is not following the [currently documented process](https://github.com/wordpress-mobile/swift-style-guide/blob/master/PROCESS.md) for style guide proposals.

I don't really mind the process, and I think just sending PRs can be fine.  But it feels like we're undermining ourselves by not being consistent with our current documentation.

We should at least update the documentation or remove it entirely.

**Comment by diegoreymendez** [comment]
PS: for clarity, the discussion I pointed out doesn't need to happen in this PR.

The requested changes are only the ones pointed out in inline comments.  Let me know when this is ready again.

**Comment by SergioEstevao** [comment]
@diegoreymendez update the document, please have a look.

**Comment by SergioEstevao** [comment]
@diegoreymendez do you have time to give it another look?

**Review by diegoreymendez — APPROVED** [review]
Sorry for the delay @SergioEstevao

**Pull request merged**